napari/benchmarks/README.md

```diff
@@ -1,3 +1,23 @@
-These are benchmarks to be run with
-[asv](https://asv.readthedocs.io/en/stable/). They are not distributed with
+# Napari benchmarking with airspeed velocity (asv)
+
+These are benchmarks to be run with airspeed velocity 
+([asv](https://asv.readthedocs.io/en/stable/)). They are not distributed with
 installs.
+
+## Example commands
+
+Run all the benchmarks:
+
+`asv run`
+
+Do a "quick" run, where each benchmark function is run only once:
+
+`asv run --environment conda -q`
```

Comment:
Would [`asv dev`](https://asv.readthedocs.io/en/stable/commands.html#asv-dev) be a shorter alternative here (implying `--python=same`), or is there a strong reason to specify to explicitly specify the `conda` environment.

Reply:
Yes, good point. I've updated the sequence of examples to show off both the `--python=same` approach and `asv dev`.